src/pages/settings/PasswordPage.js

```diff
@@ -1,25 +1,158 @@
+                            <Text style={[styles.mb6, styles.textP]}>
+                                Changing your password will update your password for both your Expensify.com
+                                and Expensify.cash accounts.
+                            </Text>
+                            <View style={styles.mb6}>
+                                <Text style={[styles.mb1, styles.formLabel]}>Current Password*</Text>
+                                <TextInput
+                                    secureTextEntry
+                                    autoCompleteType="password"
+                                    textContentType="password"
+                                    style={styles.textInput}
+                                    value={this.state.currentPassword}
+                                    onChangeText={currentPassword => this.setState({currentPassword})}
+                                />
+                            </View>
+                            <View style={styles.mb6}>
+                                <Text style={[styles.mb1, styles.formLabel]}>New Password*</Text>
+                                <TextInput
+                                    secureTextEntry
+                                    autoCompleteType="password"
+                                    textContentType="password"
+                                    style={styles.textInput}
+                                    value={this.state.newPassword}
+                                    onChangeText={newPassword => this.setState({newPassword})}
+                                />
+                            </View>
+                            <View style={styles.mb6}>
+                                <Text style={[styles.mb1, styles.formLabel]}>Confirm New Password*</Text>
+                                <TextInput
+                                    secureTextEntry
+                                    autoCompleteType="password"
+                                    textContentType="password"
+                                    style={styles.textInput}
+                                    value={this.state.confirmNewPassword}
+                                    onChangeText={confirmNewPassword => this.setState({confirmNewPassword})}
+                                />
+                            </View>
+                            {this.props.account.requiresTwoFactorAuth ? (
+                                <View style={styles.mb6}>
+                                    <Text style={[styles.mb1, styles.formLabel]}>Two Factor Code</Text>
+                                    <TextInput
+                                        style={styles.textInput}
+                                        value={this.state.twoFactorCode}
+                                        onChangeText={twoFactorCode => this.setState({twoFactorCode})}
+                                        placeholder="Required when 2FA is enabled"
```

Comment:
Can we make these inputs (at least the last 2 ones) work so that when you can also press enter instead of having to click the button? I think with `onSubmitEditing`?